### DISCUSSION TITLE
How to format Graylog streams like the one in your GREAT post?

### DISCUSSION THREAD

**Discussion by spcano01** [RobertDWhite/WhiteMatterTech#10] [Q&A]
I'm having issues with the recent 12RC5 update, and needed to standup a remote Graylog server.  It's formatting doesn't look like the one in your link or the normal unRAID Syslog.  

Looks like I need to map: 
- facility : Category
- facility_num : Level?

I ASSumed it would come over as is, but it doesn't seem that way.  Thanks for the great write up, definitely will use the Docker portion as soon as I solve my servers issues.

**Comment by RobertDWhite** [comment]
Hello! 

So the log I think you're seeing is actually from a Docker container. The docker syslog by default is formatted that way, and I have never taken the time to change the format on either GrayLog or from the logging host. Can you post a screenshot of what you're seeing currently? What service are you using to send the logs (e.g., rsyslog, etc)?
